src/AbstractQuery.php

```diff
@@ -116,4 +116,17 @@ public static function createFromRequest($data)
             ->verify()
             ->filterConstraints();
     }
+
+    private function isConstraintExists($key)
+    {
+        if (!isset($this->constraints[$key])) {
+            return false;
+        }
+
+        if ('' === $this->constraints[$key]) {
+            return false;
+        }
+
+        return true;
```

Comment:
How about using `empty`?

```suggestion
        return empty($this->constraints[$key] ?? null);
```

Reply:
I think a separate method is not needed here, because it is used in one place only and you do not need to find a proper name.